v2/anomalydetection/frontend-setup.mdx

```diff
@@ -0,0 +1,129 @@
+---
+sidebar_position: 2
+title: Frontend Setup
+hide_title: true
+---
+
+# Setting up the Frontend
+
+Setting up the frontend is relatively simple. You need to generate a request ID for each authentication event attempt. 
+This request ID is used as a parameter for the anomaly detection for detecting anomalies such as bot detection, impossible travel, and device information.
+
+Using it on other flows will not make a real difference, as, by their nature, other flows prevent this kind of anomalies.
```

Comment:
Which other flows?

Reply:
We should make it clear that:
- This frontend setup is only required for bot detection and anomaly IP based detection like impossible travel detection.
- Bot detection is only useful for email password login flows.

Reply:
done